### PR TITLE
Including tests for the rhds/opendatahub-operator v1 branches

### PR DESCRIPTION
**Pull request by crobby**

### CODE REVIEW

**Comment by anishasthana** [comment]
can you squash the commits?

**Comment by crobby** [comment]
> can you squash the commits?

Will squash once I get something that works.

**Comment by crobby** [comment]
@alvaroaleman @wking @anishasthana  ptal, thanks.

File: ci-operator/config/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  to: odh-tests
+- context_dir: .
+  dockerfile_path: build/Dockerfile.modh
+  from: os
+  to: opendatahub-operator
+operator:
+  bundles:
+  - dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/opendatahub/opendatahub-operator:v0.9.0
```
Comment (anishasthana) [inline ci-operator/config/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0.yaml:24]: shouldn't we pull 1.0.0 now?

File: ci-operator/jobs/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+  cron: 30 3 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: v1.0.0
```
Comment (anishasthana) [inline ci-operator/jobs/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0-periodics.yaml:56]: should it be 1.0.1 now?

File: ci-operator/config/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0.yaml (hunk 1 above)
Comment (crobby) [reply ci-operator/config/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0.yaml:24]: This has to do with the bundle that is included in the repo.  It's only used in the test runs, so it hasn't been updated.  It's just gets replaced with the image for the operator that is built during this test run.

File: ci-operator/jobs/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0-periodics.yaml (hunk 2 above)
Comment (crobby) [reply ci-operator/jobs/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0-periodics.yaml:56]: The branch is still the v1.0.0 branch.  Each release in the series is a new tag at the end of the branch.

**Review by anishasthana — APPROVED** [review]
/lgtm

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**